python/src/geolibre/geolibre.py

```
@@ -16,7 +16,7 @@
 import urllib.parse
 import uuid
 import warnings
-from typing import Any, Callable
+from typing import Any, Callable, Sequence
```

Ruff (0.16.1)

[warning] Import from `collections.abc` instead: `Callable`, `Sequence`

Import from `collections.abc`

(UP035)